diff -Nru extace-1.4.6/configure.in extace-1.4.6.new/configure.in
--- extace-1.4.6/configure.in	Sun Jan 21 05:08:48 2001
+++ extace-1.4.6.new/configure.in	Tue Jan 23 03:24:44 2001
@@ -34,9 +34,6 @@
 dnl AC_HEADER_STDC
 AC_CHECK_HEADERS(fcntl.h sys/ioctl.h sys/time.h unistd.h pthread.h)
 
-CFLAGS="-O2 "
-AC_SUBST(CFLAGS)
-
 EFENCE=""
 AC_SUBST(EFENCE)
 
diff -Nru extace-1.4.6/extace/Makefile.am extace-1.4.6.new/extace/Makefile.am
--- extace-1.4.6/extace/Makefile.am	Sun Jan 21 05:15:18 2001
+++ extace-1.4.6.new/extace/Makefile.am	Tue Jan 23 03:24:25 2001
@@ -2,7 +2,6 @@
 
 INCLUDES = @GTK_CFLAGS@
 
-
 noinst_PROGRAMS = sine
 
 sine_LDADD = -lm
@@ -50,8 +49,7 @@
 
 COMMON_FLAGS = @PFLAGS@ @DFLAGS@ 
 
-CFLAGS += $(COMMON_FLAGS) -Wall 
-	
+CFLAGS = @CFLAGS@ -Wall 
 
 EXTRA_DIST = extace.desktop README NEWS TODO
 
